src/implementations/twig/components/menu/menu.story.js

```diff
@@ -34,12 +40,31 @@ const getArgTypes = () => {
         type: 'text',
       },
     },
+    cta_link: {
+      name: 'call to action',
+      type: { name: 'boolean' },
+      defaultValue: false,
```

Comment:
this `defaultValue` is not needed, we now set it via the `getArgs()` which you did, so you can remove it here .. ;)